src/sparql/QueryObjectBuilder.ts

```diff
@@ -123,11 +123,19 @@ export default class QueryObjectBuilder {
 			triples: [ this.tripleBuilder.buildTripleFromQueryCondition( condition, conditionIndex ) ],
 		};
 
-		const existsReferenceFilter: FilterPattern = {
+		let referenceExistsOrNot = '';
```

Comment:
Not 100% sure about this. If I don't initialize it the `expression` in `referenceFilter` complains. Open to suggestions here.

Reply:
I think it's good TBH

Reply:
Haha. Can't believe this is approved. (Next comes refactoring)